atomic_reactor/plugins/post_import_image.py

```diff
@@ -80,12 +80,14 @@ def run(self):
 
         self.osbs = get_openshift_session(self.workflow, self.openshift_fallback)
         self.get_or_create_imagestream()
-        self.process_tags()
+
         try:
-            self.osbs.import_image(self.imagestream_name, tags=self.get_trackable_tags())
+            self.osbs.import_image_tags(self.imagestream_name, self.get_trackable_tags(),
+                                        self.docker_image_repo, insecure=self.insecure_registry)
```

Comment:
Why aren't you passing the source_registry information into osbs-client here, since atomic_reactor has the correct reactor_config and can easily access it?

Reply:
I am, with 'self.docker_image_repo'

Reply:
Hmm I see. Would it be simpler overall to have two parameters? One for the registry and one for the repo?